HedgeModManager/CodeFile.cs

```diff
@@ -43,6 +50,63 @@ public Version FileVersion
             }
         }
 
+        public List<CodeDiffResult> Diff(CodeFile old)
+        {
+            var diff       = new List<CodeDiffResult>();
+            var addedCodes = new List<string>();
+
+            foreach (var code in Codes)
+            {
+                // Added
+                if (old.Codes.Where(x => x.Name == code.Name).Count() == 0)
```

Comment:
Instead of using Count() == 0 for these, use !Any(). Count iterates through these the entire collection while Any just checks for one item. Using Count() usually doesn't matter but with LINQ you want to avoid that.